pkg/ipamd/datastore/data_store.go

```diff
@@ -200,8 +201,8 @@ func (ds *DataStore) AddIPv4AddressToStore(eniID string, ipv4 string) error {
 	ds.lock.Lock()
 	defer ds.lock.Unlock()
 
-	log.Tracef("Adding ENI(%s)'s IPv4 address %s to datastore", eniID, ipv4)
-	log.Tracef("IP Address Pool stats: total: %d, assigned: %d", ds.total, ds.assigned)
+	log.Infof("Adding ENI(%s)'s IPv4 address %s to datastore", eniID, ipv4)
+	log.Infof("IP Address Pool stats: total: %d, assigned: %d", ds.total, ds.assigned)
```

Comment:
Is there no trace option? This will cause a lot more logging, and we are not that interested in these lines by default. Even debug is too verbose here.

Reply:
Unfortunately, zap logger does not have trace level. 

Do you have any suggestions here, other than info log level ?

Reply:
Hmm, I guess `Debug` would work for the other cases, but these two I think we just need to remove. They don't add much information that is not already logged at `Info` level on line 223.

Reply:
Got it, thanks for the clarification! Debug it is then...

Reply:
Agreed. Debug is appropriate here.